core-web/libs/ui/src/lib/components/dot-copy-button/dot-copy-button.component.ts

```diff
@@ -31,20 +21,42 @@ import { DotClipboardUtil } from '../../services/clipboard/ClipboardUtil';
     imports: [TooltipModule, ButtonModule, NgIf],
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class DotCopyButtonComponent implements OnInit {
-    @Input() copy = '';
-    @Input() label: string;
-    @Input() tooltipText: string;
+export class DotCopyButtonComponent {
+    /**
+     * String to be copied to clipboard
+     */
+    copy = input('');
 
-    constructor(
-        private dotClipboardUtil: DotClipboardUtil,
-        private dotMessageService: DotMessageService,
-        private readonly cd: ChangeDetectorRef
-    ) {}
+    /**
+     * Label to be displayed in the button
+     */
+    label = input('');
```

Comment:
The last time I checked our `jest` version,  they were not supporting the new `inputs` for testing. Did we update `jest`?

Reply:
Fixed.